http/server.go

```diff
@@ -223,17 +234,7 @@ func decodeAddresses(btcRefundAddr string, lpBTCAddr string, lbcAddr string) ([]
 	return btcRefAddrB, lpBTCAddrB, lbcAddrB, nil
 }
 
-func (s *Server) getSignatureFromHash(hash string, hashBytes []byte) (string, error) {
-
-	quote, err := s.db.GetQuote(hash)
-	if err != nil {
-		return "", err
-	}
-	if quote == nil {
-		return "", fmt.Errorf("quote not found : %v", hash)
-	}
-	p := getProviderByAddress(s.providers, quote.LPRSKAddr)
-
+func (s *Server) getSignatureFromHash(p providers.LiquidityProvider, hashBytes []byte) (string, error) {
 	signature, err := p.SignHash(hashBytes)
 	if err != nil {
 		return "", err
```

Comment:
This function is not needed anymore

Reply:
why not? the server uses this function to give back the signature of the quotehash to the user.

Reply:
done.